ISSUE_TEMPLATE/doc_issue.md

```diff
@@ -0,0 +1,34 @@
+### Does the doc defect  apply to a code bug? 
+- Yes __
+  - Specify the issue for the code bug ____.
+  - Add the Doc bug label.
+- No __
+
+### Does the documentation apply to a particular release?
+- Yes ____
+  - Specify the release:____
+- No  ____
+
+### Does the documentation apply to all operating systems? 
```

Comment:
I wouldn't ask this question. Given the move to containers everything is increasingly linux, so this seems strange. We also don't have a documented list of supporting operating systems

Reply:
I removed the operating system information.